CK3toEU4/Data_Files/blankMod/output/common/countries/Empire of the North Sea.txt

```diff
@@ -0,0 +1,235 @@
+	Brisk Brott Bru Bøkesuden Capella Delfin Desideria Dragmarken Draken Draug Draupner Dromunden Dronningen Ebenezer "Eirik Blodøks"
+	Ellida Erkesuden "Erling Jarl" Falken Fenris Finnmarken Fitjarbranden Fjordbris Fjordbuen Fjorddrott Fjordknollen Fjordsol Flint
+	Forra Fredsbryter Freia Frigg Frost Frøy Frøysleiken Færdesmanden Galdalv Gammar Garm Garsøy Gjestafløyet Glommen Gor Gornen Gram
+	Gråbein Gråfellen Grågåsen Gullbringen Gyller "Haakon Adalstein" "Haakon Jarl" Habil Hadsel Haien Halfdansnautr Halten "Harald Gråfell"
+	"Harald Haarfagre" "Harald Hardraade" Haugesund Hauk Haukelid Hedmark Hegren "Heilag Halvard" "Heilag Olav" Heimdal Heinersuden
+	Hekseblakken Heljarskinn Hera Hjelmeland Hjelmelandsfjord Hjelperen Honningsvåg Hordabranden Hother Hugro Hvalrossen Hysnes Hårkniven
+	Ingøy Irma Isaksbussen Jarlsberg Jarnmeiss Jernbarden Jernland Jernskogmøya Jotunfjell Jupiter Jæren Jøsenfjord Jøsur Karlshovde Karmøy
+	Keipa Kjempen Kjertesveinen Knurr "Kong Gor" "Kong Gudrød" "Kong Haakon" "Kong Halfdan" "Kong Harald" "Kong Magnus" "Kong Nor" "Kong Olav"
+	"Kong Sverre" Kongsberg Korsfjord Kosmos Kristsuden Krokodillen Krossuden Kvalsund Kvitsøy Kyndelsmesse Kyngesbussen Lafjord Lafrantz
+	Langesund Langfredag Lillesand Lindormen Lofoten Lom Lyngen Lyra Lysebussen "Magnus Berrføtt" "Magnus Blinde" "Magnus Gode"
+	"Magnus Lagabøte" Margaretasuden Mariasuden Mars Mercur Meteor Midnatsol Midtvinter Mikkelsmess Mira Mjølner Molde Mosken Moss
+	Munkholmen Myklefløyet Mørvika Møvik Møysalen Najaden Narvik Neptun Nidaros Njord Nor Nordavind Nordhav Nordkapp Nordkyn Nordnorge
+	Nordstjernen Nornen Nova Nyholm Nøk Ognarbranden "Olav Kyrre" Olavssuden Ombofjord Oppland Orion Ormen Oslo "Ottar Jarl" "Patrioten Georg"
+	Petersbåten Pol Polarlys Postulasuden "Prins Olav" "Prinsesse Ragnhild" Providentia "Ragnvald Jarl" Raket Rakne Randaberg Ranen
+	Ranrike Rap Rauden Raudsuden Ravnanger Reinen Ringulv Risør Rogaland Rokken Ropeid Ryfylke Rygjarbranden Saksesuden Saltdal Salten
+	"Sanct Olaf" "Sanct Svithun" Sandanger Sandeid Sarp Sauda Saude Selifar Selvik Sigarsøyka "Sigurd Jorsalfare" "Sigurd Jarl" Sirius
+	Skagerak Skien Skjeggen Skjerstad Skjold Skjoldmøya Skjærtorsdag Skorpionen Skrim Skudenes Skytteren Sleipner Snar Snø Solbakk Solglimt
+	Solli Solve Stadt Stangarfare Stegg Stigandi Stjerne Storm Strand Strindaknollen Støren Sundal Sunnifusuden Sveinsnautr Svenner
+	Sværdfisken Syftesok Sygnafylke Sættarspillir Sønnavind Sørøy Søviknes Tanafjord Tarmen Tau Taurus Tegnen Terne Thermopylæ Tignesvennen
+	Titran Tjeldsund Tor Torden "Tore Jarl" Torshovde Tranen Tranen Troll Trollfjord Trondheim Trudvang Tyr Uller Ulven Ulvhalden Ulvhedningen
+	Utvær Vaagen Vale Valkyrien Vallebussen Valslynga Varangberg Vardø Vargen Varøy Vesteraalen Vestevind Vestri Vidar Vidsjå Viken Vindafjord
+	Vinland Visund Vollsøy Volva Væringen Våle Æge Æger Øibuen Ørn Årdalsfjord Årmannen
+}
```

Comment:
Rename this to be one word. (EmpireoftheNorthSea). Make sure the reference is updated as well.